src/csharp/PesterTests/FactoryTests.cs

```diff
@@ -0,0 +1,95 @@
+namespace PesterTests
+{
+    using Pester;
+    using Xunit;
+
+    /// <summary>
+    /// Factory Tests
+    /// TODO test implementations
+    /// </summary>
+    public class FactoryTests
+    {
+        [Fact]
+        /// <summary>
+        /// Test CreateNoteProperty
+        /// </summary>
+        public void Test_CreateNoteProperty()
+        {
+            var name = "test";
+            var value = "test";
+
+            var actual = Factory.CreateNoteProperty(name, value);
```

Comment:
Nothing is asserted in those tests.

Reply:
Implemented a test, further tests required here.